pkg/vadmin/start_db_vc_test.go

```diff
@@ -0,0 +1,73 @@
+/*
+ (c) Copyright [2021-2023] Open Text.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package vadmin
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	vops "github.com/vertica/vcluster/vclusterops"
+	"github.com/vertica/vertica-kubernetes/pkg/vadmin/opts/startdb"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+const TestCatalogPrefix = "/data"
+
+// mock version of VStartDatabase() that is invoked inside VClusterOps.StartDB()
+func (m *MockVClusterOps) VStartDatabase(options *vops.VStartDatabaseOptions) error {
+	// verify basic options
+	err := m.VerifyCommonOptions(&options.DatabaseOptions)
+	if err != nil {
+		return err
+	}
+
+	// verify common options
+	err = m.VerifyCommonOptions(&options.DatabaseOptions)
```

Comment:
Why do we call it twice?

Reply:
oh! This is my typo, it should be `m.VerifyInitiatorIPAndEonMode`, Please address this in your PR if you have any chance.